definitions.yaml

```diff
@@ -666,3 +666,84 @@ definitions:
       auth_token:
         type: string
         description: The newly created API token
+
+  # The list of all invitations.
+  V4InvitationsResponse:
+    type: array
+    description: List of members that belong to this organization
+    items:
+      $ref: '#/definitions/V4Invitation'
+
+  V4Invitation:
+    type: object
+    properties:
+      accepted:
+        type: string
+        description: Date/Time that the invitation was accepted
+      account_expiry_days:
+        type: integer
+        description: The number of days after which the account should expire
+      created:
+        type: string
+        description: Date/Time that the invitation was created
+      email:
+        type: string
+        description: Email of the invitee
+      invited_by:
+        type: string
+        description: Email of the person that created the invitation
+      organizations:
+        type: array
+        description: List of organizations that the invitee will be added to
+        items:
+          type: string
+      status:
+        type: string
+        description: The status of the invitation. Either 'pending', 'accepted', or 'expired'
```

Comment:
could we have revoked? or is invitation expiry short enough to not need revocation of invites?

Reply:
Currently no way to revoke an invite. We could create a DELETE endpoint on an invite, we would delete the object then, and there wouldn't be a `revoked` state left. 

I think this feature doesn't need that much of a paper trial. but that is up to debate.

Reply:
as long as we have sth like 72h expiration a delete/revoke is not needed